packages/components/src/custom-select-control/styles.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import InputBase from '../input-control/input-base';
+import { Container as InputControlContainer } from '../input-control/styles/input-control-styles';
+
+type BackCompatMinWidthProps = {
+	__nextUnconstrainedWidth: boolean;
+};
+
+const backCompatMinWidth = ( props: BackCompatMinWidthProps ) =>
+	! props.__nextUnconstrainedWidth
+		? css`
+				${ InputControlContainer } {
+					min-width: 130px;
+				}
```

Comment:
Unfortunate that we need an override, but this is only for temporary back compat, and I don't expect us needing to do this often.

Reply:
Apologies in advance because I may have forgotten this detail, but — is the plan here to follow our ["deprecating styles" strategy](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/CONTRIBUTING.md#deprecating-styles)?

Reply:
Correct! I'm working on this as part of the follow-ups for #42460, and will post the PR soon.